queries/html_tags/injections.scm

```diff
@@ -1,9 +1,10 @@
 ; <style>...</style>
 (
   (style_element
-    (start_tag
-      (tag_name) .) 
+    (start_tag) @_no_attribute
     (raw_text) @css)
+  (#match? @_no_attribute "^\\<\\s*style\\s*\\>$")
```

Comment:
you could try lua patterns (with lua-match?). Regular match uses Vim regexes which have their own quirks.

Reply:
I've done it locally.

I'm also trying locally if the queries before were faulty due to nightly or is it because of false queries, bisecting rn